src/components/QuizTemplate.tsx

```diff
@@ -132,9 +133,31 @@ const QuizTemplate: React.FC<QuizProps> = QuizProps => {
     return shuffleModalArr[0];
   };
 
-  const checkAnswer = (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+  const selectOption = (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+    // Set the selected option
+    setSelectedOption(e.currentTarget.value);
+
+    // Get answer buttons
+    const answerBtns = document.getElementsByClassName("answers-btns")
+
+    // Remove previous highlights
+    for (const btn of answerBtns) {
+      btn.classList.remove("answers-btns--selected")
+    }
+
+    // Highlight current option
+    e.currentTarget.classList.add("answers-btns--selected")
+  }
+  
+  const checkAnswer = () => {
+    // Get user's selected option
```

Comment:
I don't think this comment is needed since the next line of code is self explanatory

Reply:
Yeah sorry about that. I have a habit of commenting everything sometimes. 😅